documentation/privatebuildtesting.md

```diff
@@ -1,36 +1,51 @@
 Private runtime build testing
 =============================
 
-Here are some instructions on how to run the diagnostics repo's tests against a locally build private .NET Core runtime based on CoreCLR. These directions will work on Windows, Linux and MacOS. 
+Here are some instructions on how to run the diagnostics repo's tests against a locally built private .NET Core runtime based on CoreCLR. These directions will work on Windows, Linux and MacOS. 
 
-1. Build the runtime repo (see [Workflow Guide](https://github.com/dotnet/runtime/blob/main/docs/workflow/README.md)).
-2. Build the diagnostics repo (see [Building the Repository](../README.md)).
-3. Run the diagnostics repo tests with the -privatebuildpath option.
+1. Build the runtime repo (see [Workflow Guide](https://github.com/dotnet/runtime/blob/main/docs/workflow/README.md)) with `-configuration release -subset clr+libs`. A release build is highly recommended.
+2. Build the diagnostics repo (see [Building the Repository](../README.md)). It should be a clean build starting with no `artifacts`, `.dotnet` or `.dotnet-test` subdirectories.
+3. Run the `eng\privatebuild.cmd` or `eng/privatebuild.sh` test runtime install script. This installs and sets up to run (just) the latest test runtimes (currently 9.0) into the `.dotnet-test` directory.
```

Comment:
But it sets up for the private build testing. There is a property added to debugger.tests.versions.txt that the SOS configs use to key on it.  Where installruntimes.cmd/sh just do the full normal install. 

I guess it could be called setupprivatebuildtesting.cmd/sh but that is too long.